packages/opentelemetry-instrumentation-langchain/opentelemetry/instrumentation/langchain/callback_handler.py

```diff
@@ -585,7 +593,7 @@ def on_llm_end(
                 )
 
         _set_chat_response(span, response)
-        span.end()
+        self._end_span(span, run_id)
```

Comment:
you're right - but it won't change anything since an LLM span is a leaf span anyway